README.md

```diff
@@ -1,2 +1,4 @@
 # dummy-repo
 Dummy repository for testing things
+
+Some dummy content.
```

Comment:
Maybe the dummy content should be longer to allow for more interesting diffs in future example PRs.